_data/projects/AMPProject.yml

```diff
@@ -0,0 +1,13 @@
+name: The AMP Project (Accelerated Mobile Pages)
+desc: The AMP Project provides a framework (built on standard web technologies) to make it easy to build fast web pages.
+site: https://github.com/ampproject/amphtml/
+tags:
+- amp
+- css
+- documentation
+- html
+- javascript
+- web
+upforgrabs:
+  name: Great First Issues
+  link: https://github.com/ampproject/amphtml/milestone/25
```

Comment:
Got it; will it render as a ? in that case?

We had debated about whether to use a label or milestone for these when we first created them, and ended up with a milestone for various reasons (e.g. we want to keep track of all of our bugs in a milestone).  We may revisit that if we find it's causing a lot of problems, but for now if it will render but without a count here I think that's okay.